resources/eventing/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: eventing
+description: Kyma Component 'eventing'
+type: application
+version: 0.1.0
```

Comment:
We use version 1.0.0 for all charts for now to have a consistent presentation

Reply:
Done